### DISCUSSION TITLE
Atlas Television

### DISCUSSION THREAD

**Discussion by Playe4** [iptv-org/community#850] [Q&A]
### Channel ID

AtlasTV.gr

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Added for next release.

**Comment by freearhey** [comment]
According to our database this channel was closed on 12 April 2024.

https://iptv-org.github.io/channels/gr/AtlasTV
